sonic-xcvrd/xcvrd/xcvrd.py

```diff
@@ -1515,13 +1554,27 @@ def task_worker(self):
                             self.port_dict[lport]['cmis_state'] = self.CMIS_STATE_FAILED
                             continue
                         host_lanes_mask = self.port_dict[lport]['host_lanes_mask']
-                        self.log_notice("{}: Setting lanemask=0x{:x}".format(lport, host_lanes_mask))
+                        self.log_notice("{}: Setting host_lanemask=0x{:x}".format(lport, host_lanes_mask))
+			
+                        self.port_dict[lport]['media_lane_count'] = api.get_media_lane_count(appl)
+                        self.port_dict[lport]['media_lane_assignment_options'] = api.get_media_lane_assignment_option(appl)
```

Comment:
Since we are using it just once, can we directly get this value in get_cmis_media_lanes_mask instead of storing it in dictionary?

Reply:
That was my first thought. But then on a second thought, i decided to save it for usage in other stages if needed as the needs comes later.